e2e-tests/commands/assert-summary-list-row-change-link.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * assertSummaryListRowChangeLink
+ * Assert a summary list change link text.
+ * @param {Object} Summary list cypress selectors from pages/partials.
+ * @param {String} Field ID in the summary list to assert
+ * @param {String} expectedChangeLinkText text for the change link
+ */
+const assertSummaryListRowChangeLink = (summaryList, fieldId, expectedChangeLinkText) => {
+  if (expectedChangeLinkText) {
+    cy.checkText(
+      summaryList.field(fieldId).changeLink(),
+      expectedChangeLinkText,
+    );
```

Comment:
Should we assert the URL as well as text with `checkLink()`?

Reply:
discussed.  Changed name of function since we are testing url elsewhere